posts/2023-10-31-23.0.0.11-beta.adoc

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "New vendor metrics available in Open Liberty 23.0.0.11-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/lauracowen
+author_github: https://github.com/lauracowen
+seo-title: New vendor metrics available in Open Liberty 23.0.0.11-beta - OpenLiberty.io
+seo-description: With MicroProfile Metrics 5.0, you can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
+blog_description: With MicroProfile Metrics 5.0, you can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= New vendor metrics available in Open Liberty 23.0.0.11-beta
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+//Blank line here is necessary before starting the body of the post.
+
+The link:/[Open Liberty] 23.0.0.11-beta contains some <<mpmetrics, new vendor metrics for MicroProfile Metrics 5.0>> (along with link:/docs/latest/reference/feature/feature-overview.html[all GA features]). You can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
```

Comment:
```suggestion
The link:/[Open Liberty] 23.0.0.11-beta release includes new vendor metrics for MicroProfile Metrics 5.0, along with link:/docs/latest/reference/feature/feature-overview.html[all GA features]. You can now add metrics such as "REST Elapsed Time per Request" directly to your dashboards without needing to calculate them yourself.
```

Reply:
I had to change this for a missing feature anyway. So resolving without changing.